views/js/qtiCreator/model/mixin/editable.js

```diff
@@ -35,10 +35,12 @@ define([
 
             if (found) {
                 const parent = found.parent;
-                const response = element.getResponseDeclaration();
-                if (response) {
-                    invalidator.completelyValid(response);
-                    Element.unsetElement(response);
+                if (element.getResponseDeclaration) {
+                    const response = element.getResponseDeclaration();
+                    if (response) {
+                        invalidator.completelyValid(response);
+                        Element.unsetElement(response);
+                    }
```

Comment:
As far as I remember, this change is a part of other task & PR was reviewed https://github.com/oat-sa/extension-tao-itemqti/pull/1720 

Is it safe to include not related fix in this PR? Shouldn't they be independent?

Reply:
Yes, but this fix doesn't fully work without that one because it's also related to widgets deletion (`_removeSelf` function).
I think it should be safe as it's only one additional check for the function existance

Reply:
Okay, but please don't forget to update ADF-364 and related PR if this one is merged first.